backend/package.json

```diff
@@ -30,7 +30,8 @@
     "jsonwebtoken": "^8.5.1",
     "jwks-client": "^1.4.0",
     "knex": "^0.95.14",
-    "pg": "^8.7.1"
+    "pg": "^8.7.1",
+    "pino": "^7.6.0"
```

Comment:
I would agree with Shawn here, unless there's a specific problem you ran into with using fastify's logging?

Reply:
its the same logger. 

its brought out here because even thought it is part of the fastify package, logging shouldn't be dependent on fastify, but the other way around (if, for arguments sake, we switch back to express, the logging code does not have to change one bit, and fastify can simply be drop-in replaced [in the case of logging]).

Reply:
This might be the case, but I don't know that you can just use any logger here, because we have committed to using the functions of pino.  For example if you dropped in Winston, yes most of the functions work, but is it 100% compatible.  Also to the other point, we aren't using any external logger, we are using the one that comes with fastify, so why add the extra bit.   Too me, its ok either way... just making my point why not to...

Reply:
i get what you are saying. if we dropped in winston, i am sure we would have to do some patch-up work to make fastify accept it nicely, because it is used to using pino. but that is a decision about our app's logging facility and the features we want out of it, not about fastify.

we are using the logger from fastify indeed, but in a way that is both a) not dependent on fastify, and b) compatible with fastify and the rest of the app all at the same time. its splitting hairs, but this is the way to have an api-wide logger instance without having to reach in to fastify to get it back out (reaching inside the circle to get the logger instance is a no-no according to Uncle Bob). we are getting all of fastify's logging abilities because *it is fastify's logging facility*, just at the appropriate layer.

Reply:
re: version: all of fastify's logging features are actually just pino. all that fastify cares about for its logger at the end of the day is having the same interface as pino. so as long as pino 7 has the same interface as 6, we are good, and i have no evidence to the contrary, only in support. if we are uncomfortable of being ahead of the curve in terms of logging features, i don't mind bumping it down to 6.